guide/additional-info/hosting.md

```diff
@@ -0,0 +1,68 @@
+# Hosting your bot on a VPS
+
+A bot requires itself to be hosted in order for it to run, however, it would not be ideal to host it on your computer all day without shutting it down. That's where a **Virtual Private Server** (VPS) comes into place. A good VPS can mark a turning point for your project as you can host local databases, and it is reliable as well. A VPS suitable for your Discord bot is typically cheap, ranging from around $2-10/month.
```

Comment:
I'll have to skip on this as well, unless you change a bit less. I feel the original one is better:
> A bot needs to be run ("hosted") on a computer for it to run

I also attached a good wikipedia page about Virtual Private Server so no need to add more info

Reply:
Forgot to remove the "... for it to run" part, mb. What about the rest of the paragraph?

Reply:
> For a bot to be online at any time, you need to host it on a **Virtual Private Server** (VPS), rather than leaving it running locally on your computer. A good and reliable VPS can be a very important point in your project, as you can use it to host local databases. Generally the price of a VPS is cheap, around $2-10/month.

what about this?